src/Cardano/BM/Observer/Monadic.lhs

```diff
@@ -35,6 +35,85 @@ import           Cardano.BM.Trace (Trace, logError, logNotice, subTrace, traceNa
 Observes an |IO| action and adds a name to the logger
 name of the passed in |Trace|. An empty |Text| leaves
 the logger name untouched.
+\newline
+\par\noindent
+Microbenchmarking steps:
+\newline
+\par\noindent
+1. Create a |trace| which will have been configured
+   to observe things besides logging.
+
+\begin{spec}
+        import qualified Cardano.BM.Configuration.Model as CM
+        . . .
+        c <- config
+        trace@(ctx, _) <- setupTrace (Right c) "demo-playground"
```

Comment:
`trace` alonw would be sufficient?

Reply:
No `ctx` is used 
https://github.com/input-output-hk/iohk-monitoring-framework/blob/3ad5ef61d5cddd5846c5530945f376559b7e813a/src/Cardano/BM/Observer/Monadic.lhs#L91-L94

Reply:
and there is `c` ;-)